exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/SerializerTest.java

```diff
@@ -387,19 +392,19 @@ void prometheus004() {
                 + "monotonic_cumulative_double_sum_suffix_total{otel_scope_name=\"full\",otel_scope_version=\"version\",type=\"mcds\"} 5.0 1633950672000\n"
                 + "# TYPE non_monotonic_cumulative_double_sum gauge\n"
                 + "# HELP non_monotonic_cumulative_double_sum description\n"
-                + "non_monotonic_cumulative_double_sum{otel_scope_name=\"full\",otel_scope_version=\"version\",type=\"nmcds\"} 5.0 1633950672000\n"
+                + "non_monotonic_cumulative_double_sum_ratio{otel_scope_name=\"full\",otel_scope_version=\"version\",type=\"nmcds\"} 5.0 1633950672000\n"
```

Comment:
#nit: let's change the unit of `NON_MONOTONIC_CUMULATIVE_LONG_SUM` (which is the instrument that produces this) to something other than `1`. Its strange to an instrument which starts with an up down counter have a `_ratio` suffix, since gauges are most likely to represent things which are ratios.